docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -121,6 +123,7 @@ endif::[]
 * *Cloud region*: Select `US East, N. Virginia`.
 * *Availability zones*: Select `Multi`.
 ////
+. Use the slider to select the required instance size. Instances with more streaming units have more capacity. In {product-kafka}, you can create Kafka instances of 1 or 2 streaming units, however, instances larger than 1 streaming unit are currently available only as a Technology Preview and should not be used in production.
```

Comment:
A couple of suggestions:

I think I'd eliminate any notion of a _required_ size here. Instead, I think you can simply say "Use the slider to select an instance size. Instances with more streaming units have more capacity."